src/language/en.tsx

```diff
@@ -53,6 +56,28 @@ const en = {
     informationCardValuesscreen: 'Valuesscreen',
     informationCardCalenderscreen: 'Calenderscreen',
     informationCardBehaviouralactivation: 'Behavioural activation',
+
+    routeNames: {
+        Settings: 'Settings',
+        History: 'History',
+        Activities: 'Register activity',
+        ActivityRegistration: 'Register activity',
```

Comment:
Should the frontpage of the Activity view have the title Activities or Register activity? Both seem fine but activities is more similar to the other frontpage titles.

Reply:
I think register activity. The other havet more like a front page than activity. Better to tell the user what to do in this case.